### ISSUE TITLE
build iconify-icon in local failed

### ISSUE THREAD

**Issue by ruoru** [iconify/iconify#260]
i want to offline use iconify-icon package, my node version is 20 pnpm is 8.

the problem:
```sh
➜  icon pnpm run build:dist

> iconify-icon@1.0.8 build:dist /Users/zo/Desktop/icon
> rollup -c rollup.config.mjs


lib/script.js → dist/iconify-icon.js...
[!] RollupError: Could not resolve entry module "lib/script.js".
    at error (/Users/zo/Desktop/icon/node_modules/.pnpm/rollup@3.29.4/node_modules/rollup/dist/shared/rollup.js:353:30)
    at ModuleLoader.loadEntryModule (/Users/zo/Desktop/icon/node_modules/.pnpm/rollup@3.29.4/node_modules/rollup/dist/shared/rollup.js:26036:20)
    at async Promise.all (index 0)


 ELIFECYCLE  Command failed with exit code 1.
```

```sh
➜  icon pnpm run build:lib

> iconify-icon@1.0.8 build:lib /Users/zo/Desktop/icon
> tsc -b

src/functions.ts:9:5 - error TS2305: Module '"@iconify/core/lib/storage/functions"' has no exported member 'iconLoaded'.

9     iconLoaded,
      ~~~~~~~~~~

src/functions.ts:162:9 - error TS2353: Object literal may only specify known properties, and 'iconLoaded' does not exist in type 'IconifyExportedFunctions'.

162         iconLoaded,
            ~~~~~~~~~~

src/index.ts:113:5 - error TS2339: Property 'iconLoaded' does not exist on type 'IconifyExportedFunctions'.

113     iconLoaded,
        ~~~~~~~~~~


Found 3 errors.

 ELIFECYCLE  Command failed with exit code 1.

```

any one help?

**Comment by baboon-king** [comment]
Why not try `pnpm i @iconify/json`. 
This will give you all the icon data

**Comment by ruoru** [comment]
> Why not try `pnpm i @iconify/json`. This will give you all the icon data

1. We have a visual team
2. The company has restrictions on external network resources

**Comment by cyberalien** [comment]
You need to build other packages in monorepo. Run `pnpm run build` from root directory.

**Comment by cyberalien** [comment]
> 2. The company has restrictions on external network resources

Then maybe this icon component is not a good fit for your project? It is intended to be used when there are many possible icons that can be loaded, such as in icon browsers.

If you have a predefined set of icons that are used in your project, API loading might not be the best approach, it would be better to use something that bundles icons, such as Unplugin Icons, UnoCSS, Iconify plugin for Tailwind or just embedding SVGs.

**Comment by cyberalien** [comment]
Both `core` and `utils` packages are part of this monorepo, so building from monorepo will link to correct version.

Published version is irrelevant for that because `iconify-icon` package was not designed to be built outside of monorepo. Though I have not published updates for `core` package for a while, so I just published `2.0.2`, which should make it possible to build icon component outside of this monorepo.

**Comment by cyberalien** [comment]
Closing as resolved.
